src/pages/recurring-invoices/common/components/RecurringInvoiceSlider.tsx

```diff
@@ -273,7 +282,13 @@ export const RecurringInvoiceSlider = () => {
                         : null}
                     </span>
                     <span>&middot;</span>
-                    <Link to={`/clients/${activity.client_id}`}>
+                    <Link
+                      to={`/clients/${activity.client_id}`}
+                      disableNavigation={disableNavigation(
+                        'client',
+                        recurringInvoice?.client
+                      )}
+                    >
                       {recurringInvoice?.client?.display_name}
                     </Link>
```

Comment:
Should we maybe conditionally render the link and p tag? This way people will think the app's broken by the link looking clickable and doing nothing.

Reply:
@beganovich Yeah, I agree with you. So, we only disable navigation for the `Link` component. I have, of course, conditionally rendered the `<span>` tag instead of the `RouterLink` component when the property `disableNavigation` is true.

Reply:
I think you missed a point here. `something ? <Link> : <span>`. No need to change base Link component.

Reply:
@beganovich As we discussed on Slack, perhaps the best option is to have a component that will render the Link or Span instead of having direct logic in every place. Therefore, the implementation has now been changed.